library/scan_facts.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
Hrmm, this might be stupid, and if you disagree, that's okay.  Curious if we should rename this `test_scan_facts.py` ... or something that clearly differentiates it from any normal module.  It's a module intended for testing purposes only.  Thoughts?